scriptworker/constants.py

```diff
@@ -103,6 +103,7 @@
             "sha256:31035ed23eba3ede02b988be39027668d965b9fc45b74b932b2338a4e7936cf9",
             "sha256:7320c720c770e9f93df26f7da742db72b334b7ded77539fb240fc4a28363de5a",
             "sha256:9db282317340838f0015335d74ed56c4ee0dbad588be33e6999928a181548587",
+            "sha256:a22b90c7e16191a701760ef4f9159e86289ba598bf8ff5b22b7b94867530460d",
```

Comment:
@escapewindow Just to double check, should we have a different revision here, compared to https://reviewboard.mozilla.org/r/123212/diff/1#index_header and https://bug1350045.bmoattachments.org/attachment.cgi?id=8851262 ?